scripts/qa_crowbarsetup.sh

```diff
@@ -12,6 +12,9 @@ for script in $common_scripts; do
     source ${scripts_lib_dir}/$script
 done
 
+# not being sourced from mkcloud is a feature
+is_onhost && complain 9 "qa_crowbarsetup.sh was sourced in mkcloud"
```

Comment:
Maybe the error message could be more helpful. If I hadn't seen this PR I would be totally confused by that.